src/azure-cli/azure/cli/command_modules/acr/custom.py

```diff
@@ -396,6 +397,8 @@ def get_docker_command(is_diagnostics_context=False):
         docker_command = os.getenv('DOCKER_COMMAND')
```

Comment:
The function name is now misleading because the logic supports Podman as well. Consider renaming it to avoid confusion for future maintainers.